sql/core/src/main/scala/org/apache/spark/sql/RelationalGroupedDataset.scala

```diff
@@ -435,6 +435,29 @@ class RelationalGroupedDataset protected[sql](
           df.logicalPlan.output,
           df.logicalPlan))
   }
+
+  private[sql] def flatMapGroupsInPandas(
+      expr: PythonUDF
+  ): DataFrame = {
```

Comment:
The passed `PythonUDF` can be non-vectorized UDF too.

Reply:
Done